README.md

```diff
@@ -1,41 +1,87 @@
+# Cluster API Provider Tinkerbell
+
 ![](https://img.shields.io/badge/Stability-Experimental-red.svg)
+[![Go Reference](https://pkg.go.dev/badge/github.com/tinkerbell/cluster-api-provider-tinkerbell.svg)](https://pkg.go.dev/github.com/tinkerbell/cluster-api-provider-tinkerbell)
+[![CRD - reference](https://img.shields.io/badge/CRD-reference-2ea44f)](https://doc.crds.dev/github.com/tinkerbell/cluster-api-provider-tinkerbell)
+[![Go Report Card](https://goreportcard.com/badge/github.com/tinkerbell/cluster-api-provider-tinkerbell)](https://goreportcard.com/report/github.com/tinkerbell/cluster-api-provider-tinkerbell)
+
+<a href="https://kubernetes.io"><img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  height="100"></a>
+<a href="https://tinkerbell.org"><img src="https://raw.githubusercontent.com/tinkerbell/artwork/main/Tinkerbell-Logo-Landscape-Dark.png" height="100"></a>
 
-# Cluster API Provider Tink
+---
 
-This repository is
+> This repository is
 [Experimental](https://github.com/packethost/standards/blob/main/experimental-statement.md)
 meaning that it's based on untested ideas or techniques and not yet established
 or finalized or involves a radically new and innovative style! This means that
 support is best effort (at best!) and we strongly encourage you to NOT use this
 in production.
 
+Kubernetes-native declarative infrastructure for Kubernetes clusters on Tinkerbell.
+
 ---
 
-Cluster API Provider Tinkerbell (CAPT) is the implementation of Cluster API
-Provider for Tinkerbell.
+## What is the Cluster API Provider Tinkerbell
 
-## Goal
+The [Cluster API][cluster_api] brings declarative, Kubernetes-style APIs to Kubernetes
```

Comment:
This link does not work.

Reply:
Oops, thanks for catching that, fixed in #118